File: microservices/ingrediente/src/main/java/br/edu/infnet/ReceitaFacil/ingrediente/model/Receita.java
package br.edu.infnet.ReceitaFacil.ingrediente.model;

import java.time.LocalDateTime;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter@Setter
@NoArgsConstructor@AllArgsConstructor
public class Receita {
    private Long id;
    private String usuario;
    private String nome;
    private String preparo;
    private String figura;
    private Status status;
    private LocalDateTime createdDate;
    private String createdBy;
    private LocalDateTime lastModifiedDate;
    private String lastModifiedBy;
}

File: android/app/src/main/java/br/edu/infnet/receitafacil_microservices/model/Receita.kt
package br.edu.infnet.receitafacil_microservices.model

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize
import java.util.Date

@Parcelize
data class Receita(
    var id: Int,
    var usuario: String = "",
    var nome: String = "",
    var preparo: String = "",
    var figura: String = "",
    var custo: Float = 0.0f,
    var ingredientes: List<Ingrediente>? = null,
    var status: Status = Status.CRIADA,
    var createdDate: String = "",
    var createdBy: String = "",
    var lastModifiedDate: String = "",
    var lastModifiedBy: String = ""
): Parcelable
